matrix_sdk_base/src/store/mod.rs

```diff
@@ -404,7 +401,7 @@ impl StateChanges {
 
     /// Update the `StateChanges` struct with the given `RoomInfo`.
     pub fn add_stripped_room(&mut self, room: RoomInfo) {
-        self.invited_room_info.insert(room.room_id.as_ref().to_owned(), room);
+        self.room_infos.insert(room.room_id.as_ref().to_owned(), room);
```

Comment:
can we remove the `invited_room_info` field?

Reply:
No. The `RoomInfo` for invited rooms comes from a different source and those need to be kept separate, these are nowadays the same types but they come from different event types (stripped state events) and should never be mixed.